.golangci.yml

```diff
@@ -20,12 +19,10 @@ linters:
     - prealloc
     - revive
     - staticcheck
-    - structcheck
     - typecheck
     - unused
```

Comment:
So `deadcode`, `structcheck` and `varcheck` are now all a subset of `unused`?

Reply:
That's correct. According to the [linters documentation](https://golangci-lint.run/usage/linters/), the `unused` linter replaces the unmaintained `deadcode`, `structcheck` and `varcheck` linters.